hotness/anitya.py

```diff
@@ -215,10 +215,18 @@ def add_new_project(self, name, homepage):
         )
 
         # Try to guess at what backend to prefill...
-        for target, backend in backends.items():
-            if target in homepage:
+
+        # Start with package prefix first...
+        for prefix, backend in prefixes.items():
+            if name.startswith(prefix) and backend != None:
                 data['backend'] = backend
-                break
+
+        # And try homepage if backend was not detected based on prefix
+        if 'backend' not in data:
+            for target, backend in backends.items():
+                if target in homepage:
+                    data['backend'] = backend
+                    break
```

Comment:
Hmmm. I wonder if it'd be best to try the prefix-determined backend first and prod the location with a request, falling back to the homepage if that's an invalid URL. Although this wouldn't be perfect (plenty of projects neglect their PyPI releases, for example), it'd be even better.

I've been slowly working towards breaking the backend of Anitya out into a separate package that we can then use in the-new-hotness, so we could leverage the actual backend code in Anitya. Another thought I just had is it might be nice to have a little flag on projects in Anitya to indicate they were generated by a machine and might not be right.

Anyway, all that's to say that I think this is a reasonable default for the moment.